app/Models/User.php

```diff
@@ -1274,8 +1277,11 @@ public function trackCustomerIoEvent($eventName, $eventData)
      */
     public function calculateUserSubscriptionBadges()
     {
-        if (in_array('news', $this->email_subscription_topics)) {
-            $this->addBadge(BadgeType::get('BREAKDOWN'));
+        if (
+            in_array('news', $this->email_subscription_topics) &&
+            !in_array('news-subscription', $this->badges)
```

Comment:
I wonder if we ought to be consistent about referring to the badge names via the variable? (This way we're truly encapsulating the machine name behind this accessor.)

So here we'd say 
```suggestion
            !in_array(BadgeType::get('NEWS_SUBSCRIPTION'), $this->badges)
```